cluster/gce/addons/konnectivity-agent/konnectivity-agent-ds.yaml

```diff
@@ -55,6 +58,10 @@ __EXTRA_PARAMS__
               valueFrom:
                 fieldRef:
                   fieldPath: status.hostIP
+          ports:
+           - name: adminport
```

Comment:
what's exposed from this port? anything security-sensitive? I assume it's not secured